05-Token-Renewal/.angular-cli.json

```diff
@@ -19,8 +19,8 @@
       "testTsconfig": "tsconfig.spec.json",
       "prefix": "app",
       "styles": [
-        "styles.css",
-        "../node_modules/bootstrap/dist/css/bootstrap.min.css"
+        "../node_modules/bootstrap/dist/css/bootstrap.min.css",
```

Comment:
This error's shown when running the app:
![image](https://user-images.githubusercontent.com/15782359/38039900-4bb661ea-327c-11e8-88ba-7fe20a835e1f.png)
it's solved adding the angular2-jwt package to the package.json file.

Also, profile route doesn't validate the token properly because you can access when not logged in, only an error is shown:
![image](https://user-images.githubusercontent.com/15782359/38040276-21ae3e26-327d-11e8-83d2-61978fb69146.png)